src/test/scala/cognite/spark/v1/AssetsRelationTest.scala

```diff
@@ -367,13 +370,14 @@ class AssetsRelationTest extends FlatSpec with Matchers with ParallelTestExecuti
                 |lastUpdatedTime,
                 |0 as rootId,
                 |null as aggregates,
+                |null as labels,
```

Comment:
labels is a nullable field and I did not add this into schemas here but it is available in scala sdk. If I don't add "null as labels" in some test queries, it gives an error. As a user in transformations, I was skipping some nullable parameters in my queries. That was working there, how do we handle this when user skips some? 

I am thinking if this will cause some compatibility issues. If so, I would prefer deleting "labels" from Assets in scala-sdk and take it as a separate PR along with spark part. I don't prefer to add it to spark-datasource in this PR since it would take time (because this was the hard part of all).